Example/JustTweak.xcodeproj/project.pbxproj

```diff
@@ -437,14 +389,14 @@
 			);
 			inputPaths = (
 				"${SRCROOT}/Pods/Target Support Files/Pods-JustTweak_Example/Pods-JustTweak_Example-frameworks.sh",
-				"${BUILT_PRODUCTS_DIR}/GoogleToolboxForMac/GoogleToolboxForMac.framework",
+				"${BUILT_PRODUCTS_DIR}/GoogleUtilities/GoogleUtilities.framework",
```

Comment:
Not sure why this is part of the PR but this is alla unrelated. It shouldn't happen even with Cocoapods 1.5, I trust reverting back to 1.3.1 will fix it.